aquacomputer_d5next.c

```diff
@@ -19,6 +19,7 @@
 #include <linux/mutex.h>
 #include <linux/seq_file.h>
 #include <asm/unaligned.h>
+#include <linux/delay.h>
```

Comment:
Includes must be sorted, with the `asm` includes at the end (not sure why that is so, but that's what I've been told on the LKML.)